Makefile

```diff
@@ -189,5 +185,5 @@ clean-kustomize:
 .PHONY: test-kustomize $(addprefix test-kustomize-,$(KUSTOMIZE_OUTPUT))
 test: test-kustomize
 test-kustomize: $(addprefix test-kustomize-,$(KUSTOMIZE_OUTPUT))
-$(addprefix test-kustomize-,$(KUSTOMIZE_OUTPUT)): test-kustomize-%: _work/kustomize-$(KUSTOMIZE_VERSION)
+$(addprefix test-kustomize-,$(KUSTOMIZE_OUTPUT)): test-kustomize-%: _work/kustomize
```

Comment:
Can you restore the `-$(KUSTOMIZE_VERSION)` suffix? Without it, "make kustomize" might continue to use an old, incorrect version. The sequence of events that can lead to that is:
- run `make kustomize`
- check out some PMEM-CSI source code where `${KUSTOMIZE_VERSION}` is different
- run `make kustomize` again

Reply:
@pohly, I tried to mimic the sequence of commands as you mentioned, but couldn't reproduce the issue you pointed:
 1) run `make test-kustomize` with KUSTOMIZE_VERSION=3.4.0
 2) change to KUSTOMIZE_VERSION=3.3.0
 3) rerun `make test-kustomize` - now it downloads the v3.3.0 binary.

```
$ _work/kustomize version
{Version:kustomize/v3.4.0 GitCommit:2c9635967a2b1469d605a91a1d040bd27c73ca7d BuildDate:2019-11-12T05:00:57Z GoOs:linux GoArch:amd64}

$ make test-kustomize
curl -L https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/v3.3.0/kustomize_v3.3.0_linux_amd64.tar.gz -o /home/avalluri/work/go/src/github.com/intel/pmem-csi/_work/kustomize_v3.3.0_linux_amd64.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   626    0   626    0     0    578      0 --:--:--  0:00:01 --:--:--   579
100 8192k  100 8192k    0     0   674k      0  0:00:12  0:00:12 --:--:-- 1311k
tar xzf _work/kustomize_v3.3.0_linux_amd64.tar.gz -C _work

$ _work/kustomize version
Version: {Version:kustomize/v3.3.0 GitCommit:7050c6a7b692fdba6e831e63c7b83920ab03ad76 BuildDate:2019-10-24T17:54:30Z GoOs:linux GoArch:amd64}
```

Reply:
Never mind, I missed the dependency of `kustomize` on the version (`_work/kustomize: _work/kustomize_${KUSTOMIZE_VERSION}_linux_amd64.tar.gz`). That's different from before, but also works.